app/controllers/seeds_controller.rb

```diff
@@ -49,7 +49,7 @@ def new
     @seed = Seed.new
 
     # using find_by_id here because it returns nil, unlike find
-    @crop = Crop.find_by_id(params[:crop_id]) || Crop.new
+    @crop = Crop.find_by(id: params[:crop_id]) || Crop.new
```

Comment:
I haven't used cancan much  but i thought load_and_authorize_resource would only load the `@seed` on the seeds controller. The `@crop` and `@owner` needs to be loaded by the method that needs it.